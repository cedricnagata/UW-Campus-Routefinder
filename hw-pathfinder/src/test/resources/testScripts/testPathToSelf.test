# Test path to self

CreateGraph Graph

AddNode Graph A

AddEdge Graph A A 10.0

FindPath Graph A A