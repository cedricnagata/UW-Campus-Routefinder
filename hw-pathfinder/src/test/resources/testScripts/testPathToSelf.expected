# Test path to self

created graph Graph

added node A to Graph

added edge 10.000 from A to A in Graph

path from A to A:
total cost: 0.000